source/installation/install_from_source.rst

```diff
@@ -135,22 +134,27 @@ you are upgrading from stock syslogd, /etc/syslog.conf is probably a
 good starting point. Rsyslogd understands stock syslogd syntax, so you
 can simply copy over /etc/syslog.conf to /etc/rsyslog.conf. Note since
 version 3 rsyslog requires to load plug-in modules to perform useful
-work (more about `compatibility notes v3 <v3compatibility.html>`_). To
-load the most common plug-ins, add the following to the top of
+work.
+
+.. seealso::
+
+   :doc:`/compatibility/v3compatibility`
+
+To load the most common plug-ins, add the following to the top of
 rsyslog.conf:
 
 ::
 
- $ModLoad immark # provides --MARK-- message capability
- $ModLoad imudp # provides UDP syslog reception
- $ModLoad imtcp # provides TCP syslog reception
- $ModLoad imuxsock # provides support for local system logging
- $ModLoad imklog # provides kernel logging support
+  $ModLoad immark # provides --MARK-- message capability
+  $ModLoad imudp # provides UDP syslog reception
+  $ModLoad imtcp # provides TCP syslog reception
+  $ModLoad imuxsock # provides support for local system logging
+  $ModLoad imklog # provides kernel logging support
 
 Change rsyslog.conf for any further enhancements you would like to see.
 For example, you can add database writing as outlined in the paper
-"`Writing syslog Data to MySQL <rsyslog_mysql.html>`_\ " (remember you
-need to enable MySQL support during step 2 if you want to do that!).
```

Comment:
You didn't say anything about this one, but this is another broken link.

Reply:
... towards the end did obviously was not too concentrated on the changes ;-)